src/roadnet/roadnet.cpp

```diff
@@ -94,6 +94,10 @@ namespace CityFlow {
                     laneIndex++;
                 }
 
+                for (auto &lane : roads[i].lanes) {
```

Comment:
make it const

Reply:
The value type of drivableMap is without 'const'